src/GraphBLAS-sharp.Backend/Objects/ArraysExtentions.fs

```diff
@@ -0,0 +1,30 @@
+﻿namespace GraphBLAS.FSharp.Backend
+
+open Brahma.FSharp
+
+module ArraysExtensions =
+
+    type ClArray<'a> with
```

Comment:
It does not respect the invariant that the type must be 'a option.

Reply:
@dpanfilyonok Why must? In my opinion in may be Option<'t>, but may be not.

Reply:
If not, we won't be able to determine if the value is 0 or not.

Reply:
@dpanfilyonok But it is required only for ClVector, not for ClArray. But for ClVector it is specified that value is Option<'t>: https://github.com/YaccConstructor/GraphBLAS-sharp/pull/46/files#diff-fd9366b267db07ee1f0db1c7d4a56f1d2c91fee0be1da169aa171166eb294f87R131

Reply:
But its not specified for ClDenseVector itself. Therefore, problems can arise if we want to use a dense vector by itself.

Reply:
@dpanfilyonok  What is Cl DenseVector? I cannot find declaration of this type.